feat: Charge v1.1 (change position to risk) (#51)
fariapp/fari-games: public/documents/fari-rpgs/charge-rpg.md

Make this game your own.

> As a player, you get to choose which action you roll by simply saying what your character does. Choosing one approach versus another might influence the risk or effect of your action. The GM should tell you those details before you roll.

Reviewer: Should this use formatting on "risk" and "effect"?

```suggestion
As a player, you get to choose which action you roll by simply saying what your character does. Choosing one approach versus another might influence the `risk` or `effect` of your action. The GM should tell you those details before you roll.
```

>
> On a `6`, the PC wins and gets what they wanted. On a `4 or 5`, both the PC and the NPC have an effect on the result. On a > `1-3`, the NPC wins and the outcome results in a consequence for the PC.

## 2. GM Sets the Risk

The GM sets the `risk` of the action based on what has been established in the narrative.

> The risk is either, low, moderate or high.

Reviewer: Adding formatting for keywords:

```suggestion
The risk is either, `low`, `moderate` or `high`.
```

- When the risk is `low`, things are under control. Even if things go wrong, there is a way out.
- When the risk is `moderate`, the situation is dangerous. If you fail, there will be repercussions.
- When the risk is `high`, the odds are against you. Attempting to go against them might backfire heavily.

By default, an action roll has a `moderate` risk. You wouldn’t be rolling if there was no risk involved.

The GM uses the narrative context to determine which risk to pick.

> The action being used is another thing that might affect the risk of a roll — you are trying to convince a merchant who's in a bad mood, using `Sway` might be pretty `high risk`. `Bond` on the other hand could put you in a more `low risk` position, considering the situation.

Reviewer: Keyword formatting?

```suggestion
The action being used is another thing that might affect the `risk` of a roll — you are trying to convince a merchant who's in a bad mood, using `Sway` might be pretty `high risk`. `Bond` on the other hand could put you in a more `low risk` position, considering the situation.
```

The risk helps determine what failure looks like.

## 3. GM Sets the Effect Rating

> The risk isn't the only thing that helps resolve the end result of a roll. Even in a high risk situation, succeeding on a roll might still have a great `effect` on the narrative.

Reviewer: Keyword formatting:

```suggestion
The risk isn't the only thing that helps resolve the end result of a roll. Even in a `high risk` situation, succeeding on a roll might still have a great `effect` on the narrative.
```

In Charge, there are 3 possible effect ratings: `great`, `standard`, and `limited`. In most common scenarios, if you are rolling to overcome something, you are probably looking at generating a `standard` effect. The GM is responsible for telling you this information.

> If a heavy door is blocking your way, you could use `Finesse` to pick the lock. This might generate a `standard effect`, since it will take some time to do, but it's the `risk is low`. You could also try to use `Tinker` and wreck the door open with an explosive. This may create a `great effect`, but might be a `high risk` situation as it could create a lot of unwanted attention.

Reviewer: I'd reword this one a bit:

```suggestion
If a heavy door is blocking your way, you could use `Finesse` to pick the lock. This might generate a `standard effect`, since it will take some time to do, but the roll would be `low risk`. You could also try to use `Tinker` and wreck the door open with an explosive. This may create a `great effect`, but might be a `high risk` situation as it could create a lot of unwanted attention.
```

> ### Worse Position

Reviewer: Should this change to "increased risk" / Escalated Risk here?

```suggestion
### Escalated Risk
```

>
> - **Reduced Effect** : reduce the `effect` of the action by one level.
> - **Complication**: tick one segment per `consequence rating` on a new or existing `danger clock`.

> > - **Worse position**: increase the `risk` of future related action rolls by one level.

Reviewer: Maybe needs to be "escalated risk"

```suggestion
> - **Escalated Risk**: increase the `risk` of future related action rolls by one level.
```

Later in the campaign, when a milestone occurs, then you can think of new ways to make them truly unique using `talents`.

> > _Yann's character is a monster hunter. Consequently, he's pretty good at **Studying** and **Muscle** in general. Yann also just got a new **talent dot** and would like to excel at tracking monsters specifically. Because monsters aren't encountered that frequently in the game, he asks the GM if his talent could be "**Recuded Risk when I'm Shooting monsters**". The GM agrees. Now, Yann's character should be feared by most creatures roaming the lands._

Reviewer: "Recuded" => "Reduced"

```suggestion
> _Yann's character is a monster hunter. Consequently, he's pretty good at **Studying** and **Muscle** in general. Yann also just got a new **talent dot** and would like to excel at tracking monsters specifically. Because monsters aren't encountered that frequently in the game, he asks the GM if his talent could be "**Reduced Risk when I'm Shooting monsters**". The GM agrees. Now, Yann's character should be feared by most creatures roaming the lands._
```

>

> > _In another session, the GM realizes that Yann's talent is perhaps too generic, and could be tuned a little. He then asks if it could be rewritten as something like "**Recuded Risk when I'm Shooting monsters in the forest**" or something along those lines. Yann agrees, saying "My character lived in the forest all their life, and would probably be intimidated by big cities. I like it; let's go with this for now!"_

Reviewer: "Recuded" => "Reduced"

```suggestion
> _In another session, the GM realizes that Yann's talent is perhaps too generic, and could be tuned a little. He then asks if it could be rewritten as something like "**Reduced Risk when I'm Shooting monsters in the forest**" or something along those lines. Yann agrees, saying "My character lived in the forest all their life, and would probably be intimidated by big cities. I like it; let's go with this for now!"_
```
